.homeycompose/capabilities/operation_mode_state.json

```diff
@@ -1,8 +1,11 @@
 {
   "type": "enum",
   "title": {
-    "en": "Operation mode",
-    "fr": "Mode opérationnel"
+    "en": "Operation mode state",
+    "fr": "État du mode opérationnel",
+    "nl": "Bedrijfsmodus status",
+    "sv": "Driftläge status",
```

Comment:
"sv": "Driftläge"

Reply:
Indeed that's the offical name, but I need to differentiate from `operation_mode_zone`(https://github.com/OlivierZal/com.melcloud/pull/94/files#diff-76a8f017422629b6c50efbb10d541971b21c9a7147f200878071b247631bda50R7)

Reply:
I changed the logic and called it "Current operation"